webapp/src/main/webapp/vue-apps/links/components/view/LinksCard.vue

```diff
@@ -56,7 +56,7 @@
         v-if="showName && name"
         :class="!showIcon && 'pb-3 my-auto'"
         class="pt-3 px-1 full-width text-truncate-2 text-body">
-        {{ showName && name || '' }}
+        {{ showName && $t(name) || '' }}
```

Comment:
Please move the computing of `$t(name)` inside the computed value `name() {...}`
(same for other locations, in order to optimize rendering phase)

```suggestion
        {{ showName && name || '' }}
```